docs/data.rst

```diff
@@ -0,0 +1,146 @@
+.. _data:
+
+**********************
+Working with our Data
+**********************
+Data from the Cal-Adapt Analytics Engine can be retrieved, subsetted, visualized, and 
+exported using the *climakitae library*. 
+
+
+Retrieve and subset the data
+#############################
+In this section we will detail the various methods to retrieve and subset the catalog data. 
```

Comment:
Here I suggest we either link to the data catalog page on the website (https://analytics.cal-adapt.org/data/) or link & condense some of that info here to make this able to stand on its own (i.e. mention somewhere what kind of data is in the catalog)

Reply:
great idea!